app/code/Magento/Catalog/Test/Unit/Model/Product/Type/PriceTest.php

```diff
@@ -231,7 +231,7 @@ public function testTierPrices($priceScope, $expectedWebsiteId)
             $this->assertEquals(50, $tpRest->getExtensionAttributes()->getPercentageValue());
         }
 
-        for ($i = 0; $i < sizeof($tps); $i++) {
+        for ($i = 0, $iMax = sizeof($tps); $i < $iMax; $i++) {
```

Comment:
Could you also replace sizeof with count?
It's alias http://php.net/manual/en/function.sizeof.php